rules/windows/process_creation/win_rasautou_dll_execution.yml

```diff
@@ -0,0 +1,29 @@
+title: DLL Execution via Rasautou.exe
+id: cd3d1298-eb3b-476c-ac67-12847de55813
+description: Detects using Rasautou.exe for loading arbitrary .DLL specified in -d option and executes the export specified in -p. 
+status: experimental
+references:
+    - https://github.com/fireeye/DueDLLigence
+    - https://www.fireeye.com/blog/threat-research/2019/10/staying-hidden-on-the-endpoint-evading-detection-with-shellcode.html
```

Comment:
Could you plase add a link to the original LOLBAS page?

Reply:
Done!

Reply:
Thanks!